lib/all-badge-examples.js

```diff
@@ -1114,9 +1114,13 @@ const allBadgeExamples = [
         previewUri: '/jenkins/plugin/v/blueocean.svg'
       },
       {
-        title: 'JetBrains Plugins',
+        title: 'JetBrains IntelliJ IDEA Plugins',
         previewUri: '/jetbrains/plugin/v/9630-a8translate.svg'
       },
+      {
+        title: 'JetBrains ReSharper Plugins',
+        previewUri: '/resharper/v/ReSharper.Nuke.svg'
```

Comment:
Can you also add an example for the `/vpre` endpoint here and an example under the downloads section for the `/dt` endpoint you've added.